uint16[] id_list
float32[] input_pos_list
float32[] input_vel_list
float32[] input_torque_list
float32[] impedance_kp_list
float32[] impedance_kd_list
